tests/end-to-end/tests/buttons.test.js

```diff
@@ -0,0 +1,46 @@
+const { By, Key, until } = require('selenium-webdriver');
+const { loadBrowser } = require('../browserDrivers');
+const { startServer, DEFAULT_BASE_URL, LINKS_ROOT } = require('../linksServerRunner');
+
+let driver, process;
+
+beforeAll(async () => {
+  // Instantiate browser driver
+  driver = await loadBrowser();
+
+  // Start Links server
+  process = await startServer(`${LINKS_ROOT}/examples/webserver/buttons.links`);
```

Comment:
Can we make the example an input parameter?

Reply:
I think it makes sense to hardcode them under the current setup